### ISSUE TITLE
2.1.0.0: EDDN: approachsettlement/1 messages have Name_Localised, should be stripped.

### ISSUE THREAD

**Issue by Athanasius** [EDDiscovery/EDDLite#10]
As per https://github.com/EDDiscovery/EDDiscovery/issues/3237, now EDD**Lite** has been observed trying to send `Name_Localised` in `approachsettlement/1` messages:

```
2022-04-28 17:50:09,750 - ERROR - Gateway:274: Failed Validation "[<ValidationError: "Additional properties are not allowed ('Name_Localised' was unexpected)">]" (536, "EDDLite", "2.1.0.0", "https://eddn.edcd.io/schemas/approachsettlement/1", "-") from <source ip>
2022-04-28 18:31:47,959 - ERROR - Gateway:274: Failed Validation "[<ValidationError: "Additional properties are not allowed ('Name_Localised' was unexpected)">]" (535, "EDDLite", "2.1.0.0", "https://eddn.edcd.io/schemas/approachsettlement/1", "-") from <source ip>
2022-04-29 05:35:22,679 - ERROR - Gateway:274: Failed Validation "[<ValidationError: "Additional properties are not allowed ('Name_Localised' was unexpected)">]" (535, "EDDLite", "2.1.0.0", "https://eddn.edcd.io/schemas/approachsettlement/1", "-") from <source ip>

**Comment by Eahlstan** [comment]
Seems like Robby didn't update the Core when compiling.
It's currently on [1a2313c](https://github.com/EDDiscovery/EliteDangerousCore/tree/1a2313c08bbbb0cff4f53ebe5f88548fbc35ec27) which has [this](https://github.com/EDDiscovery/EliteDangerousCore/blob/1a2313c08bbbb0cff4f53ebe5f88548fbc35ec27/EliteDangerous/EDDN/EDDNClass.cs) version of the EDDN code. This thing is missing the code that strips localised things.

The [current version](https://github.com/EDDiscovery/EliteDangerousCore/blob/master/EliteDangerous/EDDN/EDDNClass.cs) has the required line:

`RemoveCommonKeys(message);          // remove _localised`

**Comment by robbyxp1** [comment]
Doh!

Fixed, 2.2
